frontend/src/components/object/ObjectFilters.vue

```diff
@@ -134,5 +163,9 @@ const searchTagging = async () => {
 .filterDropdown {
   min-width: 15rem;
   height: 2.45rem;
+  :deep(.p-multiselect-token) {
+    padding-bottom: 2px;
+    padding-top: 2px;
+  }
```

Comment:
Not completely sure, but it looks like the words Metadata and Tag in the empty fields are slightly below center field/div with this change.
![Screenshot 2023-07-18 at 12-02-14 BCBox - My Objects](https://github.com/bcgov/bcbox/assets/1546326/921fa3da-c618-4523-a3f8-378a1f779423)

Reply:
Yeah, hadn't seen on my zoom level but at 100% they're a bit off, will tweak